service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -1491,6 +1493,90 @@ void cloneGcsBucketDo() throws InterruptedException {
         expectedLifecycleRules, containsInAnyOrder(bucketInfo.getLifecycleRules().toArray()));
   }
 
+  @Test
+  void cloneGcsBucketTwice() throws InterruptedException {
+    ControlledGcsBucketResource resource =
+        ControlledResourceFixtures.makeDefaultControlledGcsBucketBuilder(workspace.getWorkspaceId())
```

Comment:
opt nits

opt - call this `cloneGcsBucketTwice_assertResourceLineage` ?

opt - if we're already testing this in integration test we don't need this?